airflow/providers/google/cloud/transfers/gcs_to_gcs.py

```diff
@@ -276,6 +280,21 @@ def _copy_source_with_wildcard(self, hook, prefix):
         self.log.info('Delimiter ignored because wildcard is in prefix')
         prefix_, delimiter = prefix.split(WILDCARD, 1)
         objects = hook.list(self.source_bucket, prefix=prefix_, delimiter=delimiter)
+        if not self.replace:
+            # If we are not replacing, list all files in the Destination GCS bucket
+            # and only keep those files which are present in
+            # Source GCS bucket and not in Destination GCS bucket
+
+            existing_objects = hook.list(self.destination_bucket, prefix=prefix_, delimiter=delimiter)
+
+            objects = list(set(objects) - set(existing_objects))
```

Comment:
```suggestion
            objects = set(objects) - set(existing_objects)
```
Is there any particular reason to use list?

Reply:
Testing on python3.7 I see set() - set() returns a set.
Therefore, I wanted to convert it to a list, in order to stay consistent with `objects` when replace flag is True.